kv/dist_sender_test.go

```diff
@@ -209,11 +212,17 @@ func TestSendRPCOrder(t *testing.T) {
 			Replicas: nil,
 		}
 		for i := int32(1); i <= 5; i++ {
-			addr := util.MakeRawAddr("tcp", fmt.Sprintf("%d", i))
+			addr, err := net.ResolveTCPAddr("tcp", fmt.Sprintf("localhost:%d", i))
```

Comment:
I'm going to change this back.  The problem is the way in which go handles addresses.  You can't have an unresolved net.addr, which is why we have the RawNetAddr.  But the documentation in RawNetAddr was not clear.  I'm going to send a PR later today which will change the name to UnresolvedNetAddr that will make all of this a lot clearer.